### PR TITLE
Problem: no submodule update in Makefile (fix #544)

### LINKED ISSUE #544: Problem: no submodule update in Makefile
some user may forget to submodule update

add this command
```
git submodule update --init --recursive
```

in Makefile
so that when you run
make or make install , make build
automatic git submodule update is activated

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: add make.sh for submodule update

### CODE REVIEW

**Review by tomtau — COMMENTED** [review]
I guess you can put this directly in the makefile?

**Comment by leejw51crypto** [comment]
i'll check to include it in Makefile

**Comment by leejw51crypto** [comment]
separate document Makefile
`make` will download third party automatically
for document: make document

File: Makefile [hunk 1]
```diff
@@ -220,40 +223,5 @@ release:
 ###                              Documentation                              ###
 ###############################################################################
 
-include third_party/cosmos-sdk/contrib/devtools/Makefile
```
Comment (yihuang) [inline Makefile:223]: if it's because of this line, we can just vendor that makefile.

**Review by tomtau — APPROVED** [review]

**Pull request merged**